static/templates/settings/data-exports-admin.handlebars

```diff
@@ -0,0 +1,18 @@
+<div id="data-exports" class="settings-section" data-name="data-exports-admin">
+    {{#if is_admin}}
+    <div class="alert" id="export_status" role="alert">
+        <span class="export_status_text"></span>
+    </div>
+    <form class="form-horizontal">
+        <p class="alert-word-settings-note">{{#tr this}}This tool does a <a href="https://zulipchat.com/help/export-your-organization">public export</a> of your organization's history.{{/tr}}</p>
+        <div class="grey-box">
+            <div class="wrapper">
+                <div class="settings-section-title">{{t "Export the organization's public-only data" }}</div>
+                <button type="submit" class="button rounded sea-green" id="export-data">
+                    {{t 'Export organization' }}
+                </button>
+            </div>
+        </div>
+    </form>
+    {{/if}}
```

Comment:
We should extend this widget in two keys ways:
(1) Add a widget with a list of the exports from RealmAuditLog.
(2) Live-update that list when the export is completed, via the event triggered in `notify_export_completed`

How to do both of those things correctly should be fairly clear from the "New Application Feature" tutorial, plus looking at any of our table-based widgets.  

I think with those completed, we can reasonably make this available to users.